api/renewer.go

```diff
@@ -112,144 +149,113 @@ func (c *Client) NewRenewer(i *RenewerInput) (*Renewer, error) {
 
 	renewBuffer := i.RenewBuffer
 	if renewBuffer == 0 {
-		renewBuffer = DefaultRenewerRenewBuffer
+		renewBuffer = DefaultLifetimeWatcherRenewBuffer
 	}
 
-	return &Renewer{
-		client:    c,
-		secret:    secret,
-		increment: i.Increment,
-		random:    random,
-		doneCh:    make(chan error, 1),
-		renewCh:   make(chan *RenewOutput, renewBuffer),
+	return &LifetimeWatcher{
+		client:        c,
+		secret:        secret,
+		increment:     i.Increment,
+		random:        random,
+		doneCh:        make(chan error, 1),
+		renewCh:       make(chan *RenewOutput, renewBuffer),
+		renewBehavior: i.RenewBehavior,
 
 		stopped: false,
 		stopCh:  make(chan struct{}),
+
+		errLifetimeWatcherNotRenewable: ErrLifetimeWatcherNotRenewable,
+		errLifetimeWatcherNoSecretData: ErrLifetimeWatcherNoSecretData,
 	}, nil
 }
 
+// Deprecated: exists only for backwards compatibility. Identical to
+// NewLifetimeWatcher with the exception of some error cases.
+func (c *Client) NewRenewer(i *LifetimeWatcherInput) (*LifetimeWatcher, error) {
```

Comment:
Does this method need to set the RenewBehavior to the old default?

Reply:
Nevermind, I see it is done in `Renew()`.  It may make sense to just do it here and let Renew pass through to Start.

Reply:
Or in both?

Reply:
Fixed.